src/index.html

```diff
@@ -15,8 +16,98 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
+  <!--#endregion -->
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <img
+        class="logo__image"
+        src="images/logo.png"
```

Comment:
The `alt` attribute for the logo image is 'logo_image'. Consider using a more descriptive text for better accessibility, such as 'Company Logo'.